File: src/1006.c
/*
 * 1006 - Average 2
 * Author: Otavio
 * Date: 25/04/2024
 */
#include <stdio.h>
 
int
main()
{
    double a, b, c;
    
    scanf("%lf", &a);
    scanf("%lf", &b);
    scanf("%lf", &c);
    
    double avg = a*.2 + b*.3 + c*.5;
    
    printf("MEDIA = %.1lf\n", avg);
    return 0;
}

File: src/JS/1006.js
/*
 * 1006 - Average 2
 * Author: Otavio
 * Date: 31/08/2024
 */
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

const A = parseFloat(lines.shift());
const B = parseFloat(lines.shift());
const C = parseFloat(lines.shift());

const MEDIA = (A*0.2) + (B*0.3) + (C*0.5);

console.log(`MEDIA = ${MEDIA.toFixed(1)}`);
